xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-ui/src/test/it/org/xwiki/test/ui/appwithinminutes/DBListClassFieldTest.java

```diff
@@ -75,10 +73,50 @@ public void testDisplayType()
         // Change the display type back to 'input'.
         dbListField.openConfigPanel();
         dbListField.getDisplayTypeSelect().selectByVisibleText("input");
-        assertTrue(isReadOnly(dbListField.getSizeInput()));
+        assertTrue(dbListField.isReadOnly());
         dbListField.closeConfigPanel();
         // Assert that the selected values have been preserved.
         assertEquals(Arrays.asList("AppWithinMinutes.DBList", "AppWithinMinutes.Date"),
             dbListField.getPicker().getValues());
     }
+
+    /**
+     * Tests that multiple select state is synchronized with the rest of the meta properties.
+     *
+     * @since 13.2
+     * @since 12.10.6
+     */
+    @Test
+    public void testMultipleSelect()
```

Comment:
no public here and no "test" prefix no?

Reply:
This is not a docker test.
- with junit 4, the `public` visibility is mandatory
- I'm not that sure about the `test` prefix but looking at the rest of the module, it looks like the norm (I'd be happy to rename the methods of this test if it's not the case).

Reply:
> I'm not that sure about the test prefix but looking at the rest of the module, it looks like the norm (I'd be happy to rename the methods of this test if it's not the case).

It's not the norm (even if you see it a lot of places). See https://dev.xwiki.org/xwiki/bin/view/Community/Testing/JavaUnitTesting/#HBestpractices

> Name the test methods with the method to test followed by a qualifier describing the test. For example importWithHeterogeneousEncodings(). In the few cases where it's not applicable, name after the use case being tested.

this part is valid for both junit4 and junit5. When we introduced junit5, I removed it from the junit4 doc (I moved everything still valid to the junit5 doc).

Reply:
Thanks, I naively thought the rule was only holding for docker based tests. Knowledge and PR updated.